code-collab-app/src/storage/peer_data.js

```diff
@@ -4,9 +4,11 @@ import io from 'socket.io-client';
 
 const peer_data = observable({
     //  hard coded data for now
-    session_list: [ ['test1','1'], 
-                    ['test2','2'], 
-                    ['test3','3']],
+    session_list: [
+        {id: 1, document_name: 'test1', peers: [1, 2]}, 
```

Comment:
Not that it matters here, just more of a heads up. I think the peer id's you get will be strings that look more like this: `a4bde6`

Reply:
ok sounds good lol. Yea i wasn't sure the format so I just threw in something random.